packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,263 @@
+| `topic`           | A description of the main focus of the message. Similar to the subject line of an email.         | Custom Internal Code                                                                   | In person physical with Homer Simpson on April 10th, 2023 |
+| `category`        | The type of message being conveyed. Like a tag that can be applied to the message.               | [SNOMED Codes](http://hl7.org/fhir/R4/valueset-medication-form-codes.html)             | [See below](#building-and-structuring-threads)            |
+| `partOf`          | A reference to another resource which the `Communication` is a component.                        |                                                                                        | [See below](#how-to-"tag"-or-group-threads)               |
+| `inResponseTo`    | A reference to another `Communication` resource which the current one was created to respond to. |                                                                                        | Communication/previous-communication                      |
+| `medium`          | The technology used for this `Communication` (e.g. email, fax, phone).                           | [Participation Mode Codes](http://terminology.hl7.org/CodeSystem/v3-ParticipationMode) | email                                                     |
+| `subject`         | A reference to the patient or group that this `Communication` is about.                          |                                                                                        | Patient/homer-simpson                                     |
+| `encounter`       | A reference to a medical encounter to which this `Communication` is tightly associated.          |                                                                                        | Encounter/example-appointment                             |
+| `sent`/`received` | The time that the message was either sent or received.                                           |                                                                                        | 2023-04-10T10:00:00Z                                      |
+
+## Building and Structuring Threads
+
+When building a thread, it is important to consider what type of resource should be used to group the thread together. Depending on the circumstances it makes sense to use different resources.
+
+If the messages are between a patient and a provider, you should use an `Encounter` resource to group the thread. An `Encounter` can be any interaction between a patient and provider, including various types of messages, so it is important that these are classified correctly. For more details, please see the [Representing Asynchronous Encounters](https://www.medplum.com/docs/communications/async-encounters) docs.
```

Comment:
Add another line to emphasize why using Encounter is valuable - billing. 

"Using an Encounter resource  to represent patient visits also makes it easier to submit claims to insurance to be reimbursed"